assembl/lib/frontend_urls.py

```diff
@@ -59,12 +59,15 @@ def get_timeline_for_date(discussion, date):
     """
     if isinstance(date, basestring):
         date = dateutil.parser.parse(date)
-
-    phases = sorted(discussion.timeline_phases, key=lambda p: p.start)
+    mindate = datetime.date(datetime.MINYEAR, 1, 1)
+    mindate = datetime.datetime.combine(mindate, datetime.datetime.min.time())
```

Comment:
can't you just use `mindate = datetime.datetime(datetime.MINYEAR, 1, 1)` or `mindate = datetime.datetime(datetime.MINYEAR, 1, 1, min_time.hour, min_time.minute)` ?

Reply:
`mindate = datetime.datetime(datetime.MINYEAR, 1, 1)` is exactly what you want indeed. This gives you `datetime.datetime(1, 1, 1, 0, 0)`, exactly the same thing as the two lines.